mkdocs.yml

```diff
@@ -11,6 +11,28 @@ nav:
       - Environment Variables: getting-started/environment-variables.md
       - SSO Settings & CSS API Account: getting-started/sso-settings.md
   - Using the Package:
+      - APIs & Components:
+          - getIntegration: using-the-package/apis-&-components/get-integration.md
+          - getRoles: using-the-package/apis-&-components/get-roles.md
+          - createRole: using-the-package/apis-&-components/create-role.md
+          - getRole: using-the-package/apis-&-components/get-role.md
+          - updateRole: using-the-package/apis-&-components/update-role.md
+          - deleteRole: using-the-package/apis-&-components/delete-role.md
+          - getRoleComposites: using-the-package/apis-&-components/get-role-composites.md
+          - addRoleComposite: using-the-package/apis-&-components/add-role-composite.md
+          - getRoleComposite: using-the-package/apis-&-components/get-role-composite.md
+          - deleteRoleComposite: using-the-package/apis-&-components/delete-role-composite.md
+          - getUserRoles: using-the-package/apis-&-components/get-user-roles.md
+          - assignUserRoles: using-the-package/apis-&-components/assign-user-roles.md
+          - unassignUserRole: using-the-package/apis-&-components/unassign-user-role.md
+          - getUsersWithRole: using-the-package/apis-&-components/get-users-with-role.md
+          - getIDIRUsers: using-the-package/apis-&-components/get-idir-users.md
+          - getAzureIDIRUsers: using-the-package/apis-&-components/get-azure-idir-users.md
+          - getGitHubBCGovUsers: using-the-package/apis-&-components/get-github-bcgov-users.md
+          - getGitHubPublicUsers: using-the-package/apis-&-components/get-github-public-users.md
+          - getBasicBCeIDUser: using-the-package/apis-&-components/get-basic-bceid-user.md
+          - getBusinessBCeIDUser: using-the-package/apis-&-components/get-business-bceid-user.md
+          - getBothBCeIDUser: using-the-package/apis-&-components/get-both-bceid-user.md
```

Comment:
should we consider breaking this into groups for better navigation?

Reply:
Ya I'll give that a try!